tests/optimize/test_objective.py

```diff
@@ -74,5 +74,27 @@ def tearDown(self):
         self.directory.cleanup()
         del self.directory
 
+class test_RelativeEntropy(unittest.TestCase):
+    """Unit tests for relentless.optimize.RelativeEntropy"""
+
+    def test_compute(self):
+        """Test compute method"""
+        v_obj = relentless.volume.Cube(L=3.0)
+        target = relentless.ensemble.Ensemble(T=10, V=v_obj, N={'A':1, 'B':2})
```

Comment:
Edit to add: you need to set a target RDF here! Again, not sure what the behavior is if you don't.